catppuccin.tmux

```diff
@@ -321,20 +38,26 @@ main() {
       eval "local $key"="$val"
   done < "${PLUGIN_DIR}/catppuccin-${theme}.tmuxtheme"
```

Comment:
This can be replaced with 
```suggestion
	set -o allexport
	source "${PLUGIN_DIR}/theme/catppuccin-${theme}.tmuxtheme" set
	set +o allexport
```
But that is maybe for another PR if at all

Reply:
I tried this suggestion, but tmux exists with return value 1. I am going to leave this as is for now.